tests/integration/conftest.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# SPDX-License-Identifier: MIT
+
+import os
+import yaml
+import pytest
+
+
+@pytest.fixture(scope="session")
+def test_env():
+    """Load test environment configuration
+
+    :return: Test environment configuration.
+    :rtype:  dict
+    """
+    config_file = os.getenv("Cachito_TEST_CONFIG", "test_env_vars.yaml")
```

Comment:
Can we use all upper case on the environment variable name, `CACHITO_TEST_CONFIG`?

You may also want to mention this env var in the `README.md`.

Reply:
done